docs/internals/team.txt

```diff
@@ -555,6 +555,32 @@ Daniele Procida
     .. _Django Girls: https://djangogirls.org
     .. _Potato: https://p.ota.to
 
+`Ola Sendecka`_
+    Ola started her adventure with Django when writing university project in
+    2009. After that she attended her first DjangoCon Europe and that
```

Comment:
maybe "... Europe which defined ..." or "... 2009. Her attendance at the next DjangoCon Europe defined ..."